Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/FragmentExtensions.kt

```diff
@@ -23,6 +23,9 @@ fun Fragment.popBackStack(): Boolean {
         }
         return false
     }
+    if (findNavController().backQueue.isEmpty()) {
+        findNavController().navigate(R.id.launcherActivity)
+    }
     return findNavController().popBackStack()
```

Comment:
Wouldn't if... else make sense here? Or does popBackStack still need to be executed?

Reply:
Yes, I think it would.

Reply:
It looks like this extension function is expecting a boolean return statement so I would leave it like this and not add an if/else in here.

Reply:
you can just return true in that case to indicate that navigation has happened